gap/Parser.gi

```diff
@@ -211,7 +212,16 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
                         current_line := ReadLineWithLineCount( filestream );
                         NormalizeWhitespace( current_line );
                     od;
-                    Append( filter_string, StripBeginEnd( current_line{ [ 1 .. Minimum( [ PositionSublist( current_line, "," ), PositionSublist( current_line, ");" ) ] ) - 1 ] }, " " ) );
+                    current_line_positition_for_filter := Minimum( [ PositionSublist( current_line, "," ), PositionSublist( current_line, ");" ) ] ) - 1;
+                    Append( filter_string, StripBeginEnd( current_line{ [ 1 .. current_line_positition_for_filter ] }, " " ) );
+                    current_line := current_line{[ current_line_positition_for_filter + 1 .. Length( current_line ) ]};
+                    if current_line[ 1 ] = ',' then
+                        current_line := current_line{[ 2 .. Length( current_line ) ]};
+                    elif current_line[ 1 ] = ')' then
+                        current_line := current_line{[ 3 .. Length( current_line ) ]};
+                    fi;
+                    ## FIXME: This elif case should never be reached. Instead there should be a break; if ); is reached.
```

Comment:
Hmm, so are you saying here "it never is reached in all my tests, I only left it here out of paranoia" ?
Or are you saying: "The code should be changed so that it never is reached" ?